synapse/rest/client/keys.py

```diff
@@ -283,15 +284,28 @@ def __init__(self, hs: "HomeServer"):
         super().__init__()
         self.auth = hs.get_auth()
         self.e2e_keys_handler = hs.get_e2e_keys_handler()
+        self._always_include_fallback_keys = False
```

Comment:
I kinda think it would be clearer to inline this?

Reply:
I ended up doing that in #15468 anyway, so I can uplift that commit to this PR. 👍